Not Found: /favicon.ico
Not Found: /favicon.ico
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 104, in content
    last_update_id = requst.GET.get('message_id') #request.session.get('updated_id', False)
NameError: global name 'requst' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
Internal Server Error: /content/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\SimpleChat\chat\views.py", line 122, in content
    response_data['id'] = last_updated_id
NameError: global name 'last_updated_id' is not defined
